### PR TITLE
UHM-8299 - Country distributions and content packages

### PR DESCRIPTION
**Pull request by mseaton**

### CODE REVIEW

File: openmrs-distro.properties [hunk 1]
```diff
@@ -6,9 +6,9 @@ parent.artifactId=${parentArtifact}
 parent.version=${parentVersion}
 parent.type=${parentType}
 
-config.${configArtifact}=${configVersion}
-config.${configArtifact}.groupId=${configGroup}
-config.${configArtifact}.type=${configType}
+content.${contentArtifact}=${contentVersion}
+content.${contentArtifact}.groupId=${contentGroup}
+content.${contentArtifact}.namespace=
```
Comment (mogoodrich) [inline openmrs-distro.properties:11]: what does the namespace do?  (assumedly a feature we aren't using?)

File: openmrs-distro.properties [hunk 2]
```diff
@@ -6,9 +6,9 @@ parent.artifactId=${parentArtifact}
 parent.version=${parentVersion}
 parent.type=${parentType}
```
Comment (mogoodrich) [inline openmrs-distro.properties:7]: What's the parent again?  the distro-pihemr?  assumedly there are changes to that as well?

File: openmrs-distro.properties (hunk 1 above)
Comment (mseaton) [reply openmrs-distro.properties:11]: See:  https://openmrs.atlassian.net/browse/SDK-363

File: openmrs-distro.properties (hunk 2 above)
Comment (mseaton) [reply openmrs-distro.properties:7]: Yes.  No, there are no changes to that.  See the properties these resolve to in the pom.

Comment (mogoodrich) [reply openmrs-distro.properties:7]: Where does the reference to the the top-level pihemr content pacakge get referenced then?  wouldn't that be in the distro-pihemr?

Comment (mseaton) [reply openmrs-distro.properties:7]: Oh sorry, yes these are the pihemr distro  The pihemr content package is handled in the pom of the zl content package.

**Pull request closed without merging**